src/test/java/edu/cornell/kfs/concur/ConcurTestConstants.java

```diff
@@ -0,0 +1,49 @@
+package edu.cornell.kfs.concur;
+
+public class ConcurTestConstants {

```

Comment:
Some of these constants are used by at least two different fixtures (and in multiple spots), so I defined them in this spot to cut down on duplication. Also, since my fixtures are enum-based, I don't think I can define them above the enum constants if I wanted to keep them in the same file. I also wanted to keep things consolidated here in case future SAE-to-Collector micro-test classes are added.